projects/plugins/jetpack/class.json-api-endpoints.php

```diff
@@ -1256,9 +1256,11 @@ public function generate_documentation() {
 							}
 						}
 					}
-					$type                  = '(' . implode( '|', $type ) . ')';
-					list( , $description ) = explode( ')', $description, 2 );
-					$description           = trim( $description );
+					$type = '(' . implode( '|', $type ) . ')';
+					if ( strpos( $description, ')' ) !== false ) {
```

Comment:
Could we switch to `str_contains` here, to improve readability? WordPress Core includes a polyfill for folks who do not run PHP 8+ yet.

```suggestion
					if ( str_contains( $description, ')' )  ) {
```

Reply:
Sounds good!